File: Asgard_Test/tests_Heimdall/benchmarks/corpus/taint_java/tp_sqli_concat.java
public class UserServlet extends HttpServlet {
    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws Exception {
        String name = request.getParameter("name");
        String sql = "SELECT * FROM users WHERE name = " + name;
        Statement stmt = conn.createStatement();
        stmt.executeQuery(sql);
    }
}

File: Asgard_Test/tests_Heimdall/benchmarks/corpus/taint_go/tp_sqli_concat.go
package main

import (
	"database/sql"
	"net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
	name := r.URL.Query().Get("name")
	query := "SELECT * FROM users WHERE name = " + name
	db.Query(query)
}

var db *sql.DB
